packages/valora-auth/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "celo-mobile-mainnet"
```

Comment:
Should the default be alfajores to make sure someone doesn't deploy to prod by accident?

Reply:
+1